.github/PULL_REQUEST_TEMPLATE.MD

```diff
@@ -15,3 +15,17 @@ Closes #
 **Regression Test (how to make sure this PR doesn't break old functionality)**
 
 - tests here
+
+<!-- For reviewers: do not remove -->
+
+**Things to look for during review**
+
+- [ ] Make sure all references to `iot` or `bx` class prefix is using the prefix variable
+- [ ] All major areas have a `data-testid` attribute. New test ids should have test written to ensure they are not changed or removed.
```

Comment:
Maybe add something like this since we don't have quite the same testing strategy on the Angular side
```suggestion
- [ ] (React) All major areas have a `data-testid` attribute. New test ids should have test written to ensure they are not changed or removed.
```

Everything else makes sense to me :+1: